Auth/Auth.Platform.Android/Authenticator/OIDCAuthenticator.cs

```diff
@@ -144,8 +144,7 @@ private Android.Net.Uri parseUri(Uri origin)
 
         private User GetUser(ICredential credential)
         {
-            //TODO: Implement me!! Need to decode the JWT token from credentials and return the user info
-            return User.newUser().WithFirstName("test").WithLastName("test").WithEmail("test@example.com");
+            return User.NewUserFromUnverifiedCredential(credential, keycloakConfig.ResourceId);
```

Comment:
I think this is a good candidate to be added to the builder.
Instead of 
```java
return User.NewUserFromUnverifiedCredential(credential, keycloakConfig.ResourceId);
```

we could add a method to the builder:
```java
return User.NewUser().FromUnverifiedCredential(credential, keycloakConfig.ResourceId);
```